schema/C1_LOAN_AGREEMENT.json

```diff
@@ -399,14 +399,14 @@
             "minimum": -99999999.99,
             "maximum": 999999999.99,
             "examples": [
-                10000.00
+                10000.0
             ],
             "fec_spec": {
                 "COL_SEQ": 18,
                 "FIELD_DESCRIPTION": "B.2. TOTAL BALANCE",
                 "TYPE": "AMT-12",
                 "REQUIRED": "X (conditional error)",
-                "SAMPLE_DATA": 10000.00,
+                "SAMPLE_DATA": 10000.0,
```

Comment:
This was a tricky one, but I found a solution.  So, what was happening here is that the python JSON library's .load() method parses these into float values and then .dump() spits them back out again into the json.  This process results in them being simplified to x.0 instead of x.00

Fortunately, the JSON library allows you to specify a function to use when parsing float values when loading in a json file.   We can use the Decimal method from the decimal library to preserve the .00 endings.

The [json library documentation](https://docs.python.org/3/library/json.html) explains how to do this and how to override the 'default' method of the JSON encoder so that it can handle Decimal objects when running .dump()

Unfortunately, naively converting Decimal objects to strings as suggested in the docs results in every Decimal value being surrounded with quotation marks.  I solved this by removing the quotation marks with the python regular expression library.